stock_location_orderpoint_cleanup/models/stock_location_orderpoint.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockLocationOrderpoint(models.Model):
+
+    _inherit = "stock.location.orderpoint"
+
+    def _get_moves_to_cleanup_domain(self):
+        self.ensure_one()
+        return [
+            ("location_orderpoint_id", "=", self.id),
+            ("state", "not in", ("done", "cancel")),
+            ("quantity_done", "<=", 0),
```

Comment:
Hi @rousseldenis  👋,

I hope you are enjoying your day off.

Can quantities be negative?

Reply:
I know that data can be weird...